### PR TITLE
enh(release): prepare Autodisco 20.10.3 release notes

### PR DESCRIPTION
**Pull request by omercier**
## Description

Release note for the new version

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x (master)

### CODE REVIEW

**Review by cgagnaire — COMMENTED** [review]
It looks like a note is missing in the french ones.

Try to also apply the 60-80 characters per line.

**Comment by omercier** [comment]
> It looks like a note is missing in the french ones.

Indeed! 

> 
> Try to also apply the 60-80 characters per line.
:+1:

**Review by cgagnaire — APPROVED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -123,6 +123,13 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon Auto Discovery
 
+### 20.10.3
+
+#### Security fixes
+
```
Comment (thiuyendang) [inline en/releases/centreon-commercial-extensions.md:129]: Missing mention to those fixes : 
[HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04
[Core] Creation forms generate 400 errors

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -130,6 +130,13 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon Auto Discovery
 
+### 20.10.3
+
```
Comment (thiuyendang) [inline fr/releases/centreon-commercial-extensions.md:134]: Missing mention to those fixes : 
[HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04
[Core] Creation forms generate 400 errors

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**